backend/src/controllers/questionController.js

```diff
@@ -13,9 +13,10 @@ export default class QuestionController {
       QuestionModel.insertQuestion(question, (err, result) => {
         if (err) {
           reject({ error: err });
-        }
-        resolve(result);
-      });
+        } else {
```

Comment:
else is also not needed here because err will break early if the condition is true (err == true)